### ISSUE TITLE
Wifi network selection popup is too small when tray icon is on bottom of screen

### ISSUE THREAD

**Issue by shawnl** [tbursztyka/connman-ui#6]
this happens when using connman-ui in gnome-shell/gnome 3.

**Comment by mrunion** [comment]
I have no new information to add, but wanted to echo "me too". I use Arch Linux, latest updates (no TESTING repo), Openbox, compton and nVidia driver (if any of that matters.) The QT-based cmst GUI works, though.

**Comment by djvs** [comment]
I've been fiddling with "taffybar" for xmonad. which also has GTK menus dropping out of a bar.  First, I noticed that if the taffybar was on the bottom of the screen, the menus would display the same bug!  So I moved it up to the top, and voila, both connman-ui and the taffybar menus were dropping down as they were supposed to.  

Maybe an initial vertical offset would fix this?  Detecting if it hit the bottom of the screen?
